x/eibc/client/cli/query_command_orders.go

```diff
@@ -68,11 +93,62 @@ func CmdListDemandOrdersByStatus() *cobra.Command {
 				return fmt.Errorf("failed to fetch demand orders: %w", err)
 			}
 
-			return clientCtx.PrintProto(res)
+			out, err := cmd.Flags().GetString("output")
+			if err == nil && out == "json" {
+				return clientCtx.PrintProto(res)
+			}
+
+			count := 0
+
+			for _, o := range res.DemandOrders {
+				count++
+
+				fmt.Printf(`
+-id:		%s
+  recipient:	%s
+  price:	%s
+  fee:		%s
+  rollapp_id:	%s
+  status:	%s
+  packet_key:	%s
+  fulfiller:	%s
+`,
+					o.Id, o.Recipient, parseAndFormat(o.Price.String()), parseAndFormat(o.Fee.String()), o.RollappId,
+					o.TrackingPacketStatus.String(), strings.TrimSpace(o.TrackingPacketKey), o.FulfillerAddress)
+			}
+
+			fmt.Printf("\ncount: %d; ts: %s\n", count, time.Now().String())
+
+			return nil
 		},
 	}
 
+	cmd.Flags().StringP("rollapp", "r", "", "Rollapp ID")
+	cmd.Flags().StringP("recipient", "c", "", "Recipient address")
+	cmd.Flags().StringP("type", "t", "", "Packet type")
+	cmd.Flags().StringP("denom", "d", "", "Denom")
+	cmd.Flags().StringP("fulfilled", "f", "", "Filter by fulfillment status")
+	cmd.Flags().StringP("fulfiller", "a", "", "Filter by fulfiller address")
```

Comment:
I feel like the shorthands are pretty non obvious for recipient, fulfiller, fulfilled

Reply:
what would you propose then? users are free to not use them

Reply:
I guess it's fine but maybe just dont' even given options for short ones?
Because we might add another arg later which is a better fit for a short name, but it's already taken. That would lead to either a confusing backwards compatible api, or a non backwards compatible one